docs/bare-metal-promscale-stack.md

```diff
@@ -0,0 +1,184 @@
+
+### 1. Deploying TimescaleDB
+
+#### Build and install TimescaleDB
+Follow the steps on timescaledb installation (instructions for other platforms available [on the TimescaleDB install page](https://docs.timescale.com/latest/getting-started/installation))
+
+**If you don't already have PostgreSQL installed**, add PostgreSQL's third party repository to get the latest PostgreSQL packages (if you are using Ubuntu older than 19.04):
+```
+# `lsb_release -c -s` should return the correct codename of your OS
+echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -c -s)-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+sudo apt-get update
+```
+
+Next, add TimescaleDB's third party repository and install TimescaleDB, which will download any dependencies it needs from the PostgreSQL repo:
+```
+# Add our PPA
+sudo add-apt-repository ppa:timescale/timescaledb-ppa
+sudo apt-get update
+
+# Now install appropriate package for PG version
+sudo apt install timescaledb-postgresql-12
+```
+#### Configure TimescaleDB
+Configure your database using `timescaledb-tune`. This will ensure that TimescaelDB is properly added to the parameter `shared_preload_libraries` as well as offer suggestions for tuning memory, parallelism, and other settings:
+```
+sudo timescaledb-tune
+
+# Restart PostgreSQL instance
+sudo service postgresql restart
+```
+
+#### Making Promscale Extension Available (optional)
+
+The Promscale extension contains support functions to improve performance of Promscale. While Promscale will run without it, adding this extension will support it to perfrom better. 
+
+Follow the steps to compile promscale extension from source available on [Promscale extension page](https://github.com/timescale/promscale_extension).
+
+### 2. Deploying Promscale
+
+Download Promscale
+
+```
+LATEST_VERSION=$(curl -s https://api.github.com/repos/timescale/promscale/releases/latest | grep "tag_name" | cut -d'"' -f4)
+curl -L -o promscale "https://github.com/timescale/promscale/releases/download/${LATEST_VERSION}/promscale_${LATEST_VERSION}_Linux_x86_64"
+```
+
+Grant executable permissions to Promscale:
+
+```
+chmod +x promscale
+```
+
+To deploy Promscale, run the following command:
+```
+./promscale --db-name <DBNAME> --db-password <DB-Password> --db-ssl-mode allow
+```
+Note that the flags `db-name` and `db-password` refer to the name and password of your TimescaleDB database from Step 1.
+
+Furthermore, note that the command above is to deploy Promscale with SSL mode as optional. To deploy Promscale with SSL mode enabled, configure your TimescaleDB instance with ssl certificates and set the `--db-ssl-mode` flag to `require`.  Promscale will then force authentication via SSL.
+
+### 3. Deploying Prometheus
+
+**Note: Replace the versions of Prometheus listed here with the latest available version numbers**
```

Comment:
should we use the same latest trick as above instead and get the latest version number into an environmental variable, which we plug into the commands below?

Reply:
I too liked that way.